scripts/pageReader.js

```diff
@@ -0,0 +1,22 @@
+// From a given config object, scrape the page for values related to metadata fields
+// Key is field ID, Object is scrapping configuration options for that field
+function scrapeFields(scrapeConfig, returnFunc) {
+  const metadataFieldValues = {};
+
+  Object.keys(scrapeConfig).forEach((fieldId) => {
+    const config = scrapeConfig[fieldId];
+    metadataFieldValues[fieldId] = $(config.selector).html();
```

Comment:
Also, let's be careful about handling when it doesn't find anything b/c websites change all the time.